src/test/java/be/syntra/devshop/DevshopBack/services/utilities/StarRatingMapperTest.java

```diff
@@ -41,4 +44,24 @@ void canMapToStarRatingTest() {
         assertThat(result.getRating()).isEqualTo(rating.getRating());
         assertThat(result.getUserName()).isEqualTo(rating.getUserName());
     }
+
+    @Test
+    void canMapToStarRatingSet() {
+        // given
+        Set<StarRating> setOfStarRatings = Set.of(
```

Comment:
there's a util -> createRatingList

Reply:
replaced and renamed util to createRatingSet()